app/renderer/css/preference.css

```diff
@@ -324,4 +337,53 @@ i.open-tab-button {
 #open-create-org-link:hover {
 		color: #005580;;
 		text-decoration: underline;
+}
+
+.toggle {
+  position: absolute;
+  margin-left: -9999px;
+  visibility: hidden;
+}
+.toggle + label {
+  display: block;
+  position: relative;
+  cursor: pointer;
+  outline: none;
+  user-select: none;
+}
+
+input.toggle-round + label {
+  padding: 2px;
+  width: 50px;
+  height: 25px;
+  background-color: #dddddd;
+  border-radius: 25px;
+}
+input.toggle-round + label:before,
+input.toggle-round + label:after {
+  display: block;
+  position: absolute;
+  top: 2px;
+  left: 2px;
+  bottom: 2px;
+  content: "";
+}
+input.toggle-round + label:before {
+  right: 1px;
+  background-color: #f1f1f1;
+  border-radius: 25px;
+  transition: background 0.4s;
+}
+input.toggle-round + label:after {
+  width: 25px;
+  height: 25px;
+  background-color: #fff;
+  border-radius: 100%;
+  transition: margin 0.4s;
+}
+input.toggle-round:checked + label:before {
+  background-color: #8ce196;
```

Comment:
Should we change it to `#388E3C` so that it matches with the button color?

![image](https://user-images.githubusercontent.com/2263909/30547500-1124e14c-9cad-11e7-8089-f2c427f7deea.png)


![image](https://user-images.githubusercontent.com/2263909/30547452-ebbef2ee-9cac-11e7-97bc-486c433f7b89.png)

![image](https://user-images.githubusercontent.com/2263909/30547543-37bd4b64-9cad-11e7-8da1-7c0b059a32bc.png)

Reply:
Use - `#4EBFAC` or `hsl(170,47%,53%)` for both (button, switch) so that it mathces with Zulip's theme.

Make it look like - 

![image](https://user-images.githubusercontent.com/2263909/30551162-3af28b78-9cb7-11e7-9d8d-095fdf98d25d.png)

https://zulipchat.com/plans/

![image](https://user-images.githubusercontent.com/2263909/30551189-57a81738-9cb7-11e7-9411-163c9de050c1.png)